shell/detail/catalog.cattle.io.app.vue

```diff
@@ -125,9 +125,12 @@ export default {
       {{ t('catalog.app.section.lastOperation') }}: ( {{ latestOperation.status.action }} ) - <a @click="latestOperation.openLogs()">  {{ t('catalog.app.section.openLogs') }}</a>
     </span>
 
-    <Tabbed
```

Comment:
We only show counts within ResourceTabs. This was the only place I noticed where we didn't use ResourceTabs in a detail page.